zhmcclient/_utils.py

```diff
@@ -73,7 +73,7 @@
 # For CPCs in any  mode and resources independent of CPCs:
 RC_CONSOLE = 'console'
 RC_CPC = 'cpc'
-RC_PASSWORD_RULE = 'password-rule'
+RC_PASSWORD_RULE = 'password-rule'  # nosec B105
```

Comment:
I assume B105 here ... is something else than the B101 that you used in the Makefile above?

Reply:
Yes. "nosec" is for bandit, "pylint" for pylint, and "noqa" for flake8.